src/aggregate/expr/expression.c

```diff
@@ -46,21 +46,23 @@ static int evalFunc(ExprEval *eval, const RSFunctionExpr *f, RSValue *result) {
 
 static int evalOp(ExprEval *eval, const RSExprOp *op, RSValue *result) {
   RSValue l = RSVALUE_STATIC, r = RSVALUE_STATIC;
-  int rc = EXPR_EVAL_ERR;
 
-  if (evalInternal(eval, op->left, &l) != EXPR_EVAL_OK) {
-    goto cleanup;
-  }
-  if (evalInternal(eval, op->right, &r) != EXPR_EVAL_OK) {
+  if (evalInternal(eval, op->left, &l) != EXPR_EVAL_OK ||
```

Comment:
Will give priority to left operand,
so, for example,
`nan` and `nil` will return `nan`
`num` and `nil` will return `nil`

Maybe better to give priority to whether one of them is `nil` => return `nil`
Otherwise if one of them is `nan` => return `nan`